components/formats-bsd/src/loci/formats/in/ICSReader.java

```diff
@@ -1477,7 +1478,7 @@ else if (labels.equalsIgnoreCase("x y t")) {
     MetadataTools.populatePixels(store, this, true);
 
     // populate Image data
-
+    imageName = imageName.substring(imageName.lastIndexOf(File.separator) + 1);
```

Comment:
Will this work if the file separators in `imageName` don't match the system file separator?